### PR TITLE
RSDK-2867-Make Classification Labels Visible

### PR DESCRIPTION
**Pull request by kharijarrett**

### CODE REVIEW

**Comment by kharijarrett** [comment]
The fix here is to _not_ check for a specific term in the metadata to stuff the labelPath into. Instead we'll stuff that labelPath into the first output every time.  We can also do that blindly (in the absence of metadata) so that's cool.

File: services/vision/mlvision/detector.go [hunk 1]
```diff
@@ -92,8 +93,7 @@ func attemptToBuildDetector(mlm mlmodel.Service) (objectdetection.Detector, erro
 				utils.Clamp(locations[4*i+getIndex(boxOrder, 2)], 0, 1)*float64(origW),
 				utils.Clamp(locations[4*i+getIndex(boxOrder, 3)], 0, 1)*float64(origH)
 			rect := image.Rect(int(xmin), int(ymin), int(xmax), int(ymax))
-			labelNum := int(categories[i])
-
+			labelNum := int(utils.Clamp(categories[i], 0, math.Inf(1)))
```
Comment (bhaney) [inline services/vision/mlvision/detector.go:96]: do you actually want to clamp to infinity? You can also specify the biggest Int possible instead `math.MaxInt`

**Review by bhaney — APPROVED** [review]
Nice, just one change!

File: services/vision/mlvision/detector.go (hunk 1 above)
Comment (kharijarrett) [reply services/vision/mlvision/detector.go]: Changed to MaxInt ty

**Pull request merged**